src/Dfc.CourseDirectory.Core/DataStore/Sql/QueryHandlers/OpenData/GetLiveCourseProvidersReportHandler.cs

```diff
@@ -29,6 +29,7 @@ public async IAsyncEnumerable<LiveCourseProvidersReportItem> Execute(SqlTransact
                 SELECT      DISTINCT c.ProviderId FROM [Pttcd].[FindACourseIndex] c
                 WHERE       c.Live = 1
                 AND         (c.FlexibleStartDate = 1 OR c.StartDate >= '{query.FromDate:MM-dd-yyyy}')
+                and c.CourseId is not null
```

Comment:
Could we not filter on OfferType = 1 instead? 
![image](https://user-images.githubusercontent.com/28952958/214893932-ffc819ad-cff2-4a3f-abd2-764cbab14ed2.png)